socialsharing_diaspora/css/socialsharing_diaspora.css

```diff
@@ -1,3 +1,14 @@
-.icon-social-diaspora {
- background-image: url('../img/diaspora.svg');
+.icon-social-diaspora,
+[data-themes*=light] .icon-social-diaspora {
+	background-image: url('../img/diaspora.svg');
+}
+
+[data-themes*=dark] .icon-social-diaspora {
```

Comment:
Ah, nice workaround :)
This `[data-themes*=dark]` + `@media (prefers-color-scheme: dark) {` seems to replace the hack we use everywhere :D

Reply:
Don't use it!
That's what the `dist/icons.css` do in server.

Reply:
> the hack we use everywhere

Please define "everywhere" ? :see_no_evil: 
This should really not be done :(

Reply:
Well not sure which is better? :P

https://github.com/nextcloud/spreed/blob/b526931253c441071d8099d158b916ca2f4744c0/src/utils/isDarkTheme.js#L28-L32

used to load room avatars with the correct background:
https://github.com/nextcloud/spreed/blob/b526931253c441071d8099d158b916ca2f4744c0/src/components/NewMessage/NewMessage.vue#L733

Reply:
Alright, not too much used I guess
https://github.com/search?q=repo%3Anextcloud%2Fspreed%20isDarkTheme&type=code